tests/functional/test.cli.run.js

```diff
@@ -9,14 +9,16 @@ import {
 const EXPECTED_MESSAGE = 'Fake Firefox binary executed correctly.';
 
 describe('web-ext run', () => {
-  it('should accept: --no-reload --source-dir SRCDIR --firefox FXPATH',
+  it('should accept: --no-reload' +
```

Comment:
@aniketkudale we use the functional test suite only to test that we don't introduce some classes of regressions that are harder to test properly using the unit tests, for this change we can probably use just an additional test in the `tests/unit/test.program.js` file (one additional test similar to this one: https://github.com/mozilla/web-ext/blob/master/tests/unit/test.program.js#L357-L369).